src/language-css/printer-postcss.js

```diff
@@ -401,6 +401,7 @@ function genericPrint(path, options, print) {
       const parts = [];
       const insideURLFunction = insideValueFunctionNode(path, "url");
 
+      let insideInSCSSSinterpolationInString = false;
```

Comment:
`SCSSS` → `SCSS`

Reply:
@lydell But the name means exactly what it contains :smile:

Reply:
@lydell let's do this